### DISCUSSION TITLE
建議字的顯示次序

### DISCUSSION THREAD

**Discussion by siukau** [rime/home#1480] [Q&A]
我剛下載了配合win11用的最新版本，使用粵拼輸入法時，打入字音後，顯示的建議中文字，是以聲調（1-6）的次序，不是以先出常用的字，令選找字非常困難。請問如何解決？

**Comment by gaboolic** [comment]
dict文件里面，有
name: xxx
version: "2020.02.08"
sort: by_weight 

你看下sort 的值，改成by_weight 就是按字频。但需要dict文件里面有字频

**Reply by siukau** [reply]
請問 gaboolic :  在那裏可找到 “dict文件”？

**Reply by gaboolic** [reply]
你看看你的粵拼輸入法的方案里面，使用的词典文件是哪个，就是哪个
看看里面怎么配置的
